assets/js/googlesitekit-modules-analytics.js

```diff
@@ -19,4 +19,4 @@
 /**
  * Internal dependencies
  */
-import 'assets/js/modules/analytics/datastore';
+import 'assets/js/modules/analytics';
```

Comment:
Let's use a relative import here 🙂

Reply:
This will be changed via #1135 anyway, but sure :)